CHANGELOG.md

```diff
@@ -11,6 +11,63 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.9.0 - 2020-08-13
+
+### Added
+
+- Add [flow exporter] feature. [Alpha - Feature Gate: `FlowExporter`]
+   * Support sending flow records using the IPFIX protocol from each Agent ([#825](https://github.com/vmware-tanzu/antrea/pull/825) [#984](https://github.com/vmware-tanzu/antrea/pull/984), [@srikartati])
+   * Add reference cookbook to visualize exported flows using Elastic Stack ([#836](https://github.com/vmware-tanzu/antrea/pull/836), [@zyiou])
+- Support [OVS hardware offload] for Pod networking: Pods can now be assigned an SR-IOV Virtual Function. ([#786](https://github.com/vmware-tanzu/antrea/pull/786), [@moshe010])
```

Comment:
an -> a

Reply:
I think it's a vowel sound, and therefore "an"

Reply:
SR-IOV is vowel?

Reply:
Ok. Seems you are right.